.github/workflows/reusable_cve_scan.yml

```
@@ -0,0 +1,211 @@
+          python3 -m venv env 
+          source env/bin/activate
+          pip install requests tqdm
+          
+      - name: Install Trivy
+        if: ${{ inputs.run_trivy }}
+        run: |
+          curl -sfL https://raw.githubusercontent.com/aquasecurity/trivy/main/contrib/install.sh | sh -s -- -b $(pwd)/trivy latest
+
+      - name: Install Grype
+        if: ${{ inputs.run_grype }}
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2046:warning:1:90: Quote this to prevent word splitting

(shellcheck)